fiftyone/migrations/revisions/v0_13_0.py

```diff
@@ -36,7 +36,7 @@ def down(db, dataset_name):
         config = run_doc.get("config", {})
         config_cls = config.get("cls", "")
         if config_cls == "fiftyone.brain.similarity.SimilarityConfig":
-            config.pop("model")
-            config.pop("metric")
+            config.pop("model", None)
```

Comment:
This shouldn't ever be important, but I happened to be working with a dataset in a broken state where this was needed, so I thought I'd throw it in anyway.

I prefer to write the migrations as fail-safe as possible